### DISCUSSION TITLE
When is the PUBLISHED status reached?

### DISCUSSION THREAD

**Discussion by fernarr** [OP-TED/eForms-SDK#708] [Q&A] [answered]
Dear TED Members,

The definition of PUBLISHED status in eNotices2 is **The notice is published online at TED.**.

From my point of view, this means that a businessID (notice ID + version ID) only gets that status after its publication on TED (09:00 CET).

However, in the preview environment, I can see that the status is PUBLISHED and the Publish Date parameter is a future date. For example, in this preview environment example, the status is PUBLISHED, the updated date is 2023-10-05T13:05:32Z", but the publish date is in the future, 2023-10-06,

```json
{
    "id": "09c3a4d2-4856-4a9d-87b6-0f4cc1edfab8",
    "versionId": "01",
    "procedureId": "d3476494-9673-4e7f-8478-1ca02e2badd9",
    "status": "PUBLISHED",
    "updatedAt": "2023-10-05T13:05:32Z",
    "submittedAt": "2023-10-05T07:11:51Z",
    "publicationId": "00094626-2023",
    "publicationDate": "2023-10-06",
    "noticeType": "2",
    "legalBasis": "32014L0025",
    "languages": ["SPA"],
    "noticeAuthor": "XXXXX"
  }
```

I think it is due to the PUBLICATION status which is not implemented yet.

Please could you confirm if the PUBLISH status is just after publishing (09:00)?

Thank you.

**Comment by YvesJo** [comment, marked as answer]
For this non-technical question, please get in touch with the helpdesk ted@publications.europa.eu

**Comment by ferrakaop** [comment]
See also the information about the Preview environment: https://docs.ted.europa.eu/home/eforms/preview/index.html#_known_enotices2_api_issues
